test/integration/edition_history_test.rb

```diff
@@ -27,6 +27,13 @@ class EditionHistoryTest < JavascriptIntegrationTest
       assert_equal ["fourth", "fifth", "sixth", "link http://www.some-link.com"], @guide.actions.map(&:comment)
     end
 
+    should "direct the user to view a published edition on GOV.UK directly, not draft" do
+      visit "/editions/#{@answer.id}"
+      click_on "History and notes"
+
+      assert page.has_content?("View this on the GOV.UK website")
```

Comment:
Could we scope this to the specific element. Conceivably this could appear on the page multiple times? (thinking of the action bar at the bottom)